### ISSUE TITLE
Q: Monitoring loss over each audio

### ISSUE THREAD

**Issue by senbukai0203** [espnet/espnet#4819]
Hi Team,
Is there any way to check training loss on each training data in espnet2? Thanks
